resources/keywords/dbaas_policy.resource

```diff
@@ -100,16 +103,91 @@ Click If Element Present
     END
 
 User Creates DBaaSPolicy For ${namespace} From Admin View As ${DBaaSPersona}
+    Set Test Variable   ${operatorNamespace}    ${namespace}
     User Navigates To DBaaSPolicy Screen    ${namespace}
-    Click If Element Present    ${lblOperand}
-    Wait Until Page Contains Element    ${btnCreateDBaaSPolicy}
-    Click Element    ${btnCreateDBaaSPolicy}
-    Create Custom DBaaSPolicy as ${DBaaSPersona}
+    User Creates DBaaSPolicy For ${DBaaSPersona}
 
 User Creates DBaaSPolicy For ${namespace} From Invalid Namespace Alert on Dev View As ${DBaaSPersona}
+    Set Test Variable   ${operatorNamespace}    ${namespace}
     User Navigates To Import MongoDB Provider Account On Developer View
     Wait Until Page Contains Element    ${errCreateDBaaSPolicy}          timeout=20s
     Click Element    ${errCreateDBaaSPolicy}
     Create Custom DBaaSPolicy as ${DBaaSPersona}
     User Selects Administrator View
 
+User Creates DBaaSPolicy For ${DBaaSPersona}
+    Click If Element Present    ${lblOperand}
+    Wait Until Page Contains Element    ${btnCreateDBaaSPolicy}
+    Click Element    ${btnCreateDBaaSPolicy}
+    Create Custom DBaaSPolicy as ${DBaaSPersona}
+
+Max Quota Exceeded Error Displays
```

Comment:
Can we change it to 'Max Quota Exceeded Error Displayed'?

Reply:
@ikanias As per Gherkin good practice, we try to maintain present tense wherever possible. IMO, I hope this is fine.

Reply:
What I meant is actually 'Error is being displayed'/ 'Error is displayed' in present tense as passive. What is written here is that the error 'displays' something (active) which is wrong :-)

Reply:
Updated phrasing